src/commands/dev/lib/dev-server.ts

```diff
@@ -142,15 +142,15 @@ export default class DevServer {
       this.output.debug(`Files changed: ${filesChangedArray.join(', ')}`);
       this.output.debug(`Files removed: ${filesRemovedArray.join(', ')}`);
       for (const [result, [requestPath, match]] of needsRebuild) {
-        if (await shouldServe(match, this.files, requestPath)) {
+        if (requestPath === null || await shouldServe(match, this.files, requestPath)) {
           this.triggerBuild(
             match,
             requestPath,
             null,
             result,
             filesChangedArray,
             filesRemovedArray
-          ).catch(err => {
+          ).catch((err: NodeJS.ErrnoException) => {
```

Comment:
Why this change?

Reply:
@TooTallNate It failed and said it had `any` type